profiles/schema.py

```diff
@@ -201,6 +201,14 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class SensitiveDataNode(DjangoObjectType):
+    class Meta:
+        model = SensitiveData
+        fields = ("ssn",)
+        filter_fields = []
```

Comment:
why does it need an empty `filter_fields` list? 🤔

Reply:
Good question 🤔. Can't really remember anymore but it had something to do with the Relay. I thought that all the other nodes has that as well but there's actually a couple of those that don't. I think I file a ticket of this and check if the relay filtering works ok without these empty lists. If it does I think each of those entries should be removed.

Reply:
if you use `DjangoFilterConnectionField`, then you need to specify which fields are `filter_fields`. otherwise it is not needed.

Reply:
Oh yeah, that was it 👍 Removing it then.